src/kademlia/dht_storage.cpp

```diff
@@ -112,7 +112,7 @@ namespace {
 		int const size = int(buf.size());
```

Comment:
If size becomes size_t here instead, maybe you get fewer casts

Reply:
OK

Reply:
well, not really, because `item.size` is `int`, it's about the same, but I can use `buf.size()` if you think it looks better, you tell me